### PR TITLE
Update hmr-with-fast-refresh.md

### PR DESCRIPTION
**Pull request by mwarger**
minor updates of punctuation, syntax, typos

### CODE REVIEW

File: pages/docs/basic-features/hmr-with-fast-refresh.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ authors:
...
+**HMR** (Hot Module Replacement) allows you update Aleph.js application without refreshing the page during **development**, any change instantly updates in the browser.
 
 ## How It Works
 
-The built-in **HMR** module of Aleph.js will create a **WS** connection to the dev server on the client during **development**, and the server will listen the app file changes. Once the HMR client gets a update signal from server, Aleph.js will re-import the updated module, for a CSS moudule the old style will be removed after the new one was appied, for a React Component the Fast Refresh will re-render the component view without losing component state.
+The built-in **HMR** module of Aleph.js will create a **WS** connection to the dev server on the client during **development**, and the server will listen the app file changes. Once the HMR client gets a update signal from server, Aleph.js will re-import the updated module. For a CSS module, the old style will be removed after the new one was appied. For a React Component, the Fast Refresh will re-render the component view without losing component state.
```
Comment (jldillman) [inline pages/docs/basic-features/hmr-with-fast-refresh.md:13]:
```suggestion
The built-in **HMR** module of Aleph.js will create a **WS** connection to the dev server on the client during **development**, and the server will listen for app file changes. Once the HMR client gets an update signal from the server, Aleph.js will re-import the updated module. For a CSS module, the old style will be removed after the new one is applied. For a React Component, the Fast Refresh will re-render the component view without losing component state.
```

File: pages/docs/basic-features/hmr-with-fast-refresh.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ authors:
 
 # HMR with Fast Refresh
 
-**HMR** (Hot Module Replacement) allows you update Aleph.js application without refreshing the page during **development**, any change instant updates in the browser.
+**HMR** (Hot Module Replacement) allows you update Aleph.js application without refreshing the page during **development**, any change instantly updates in the browser.
```
Comment (jldillman) [inline pages/docs/basic-features/hmr-with-fast-refresh.md:9]:
```suggestion
**HMR** (Hot Module Replacement) allows you update Aleph.js application without refreshing the page during **development**.  Any change instantly updates in the browser.
```

**Comment by jldillman** [comment]
@mwarger Made a few suggested changes to further increase readability.

File: pages/docs/basic-features/hmr-with-fast-refresh.md [hunk 3]
```diff
@@ -21,12 +21,12 @@ Aleph.js supports full HMR out-of-the-box for the following served files:
 
 ## What Is Fast Refresh
 
-> It's a reimplementation of "hot reloading" with full support from React. It's originally [shipping for React Native](https://twitter.com/dan_abramov/status/1169687758849400832) but most of the implementation is platform-independent. The plan is to use it across the board — as a replacement for purely userland solutions (like react-hot-loader). [¹]
+> It's a re-implementation of "hot reloading" with full support from React. It originally [shipped for React Native](https://twitter.com/dan_abramov/status/1169687758849400832), but most of the implementation is platform-independent. The plan is to use it across the board as a replacement for purely userland solutions (like react-hot-loader). [¹]
```
Comment (ije) [inline pages/docs/basic-features/hmr-with-fast-refresh.md:24]: this quote comes from *dan abramov*, please keep it

**Review by jldillman — APPROVED** [review]

**Comment by ije** [comment]
@mwarger @jldillman thank you guys!

**Pull request merged**